queries.sql

```diff
@@ -0,0 +1,28 @@
+-- Animals whose name ends with "mon"
+SELECT * from animals WHERE name LIKE '%mon';
```

Comment:
- [ ] **Good job so far 👏 👏** However, it was suggested in the project requirements to add a screenshot of all query results to the pull request description. 😂 Kindly, add them as the result of all your queries 👍
 
![image](https://user-images.githubusercontent.com/102757126/228016959-bf07ba38-a9b5-4547-9e48-4390f1638a83.png)